traefik/templates/rbac/clusterrole.yaml

```diff
@@ -31,6 +33,7 @@ rules:
       - extensions
     resources:
       - ingresses/status
+      - networking.k8s.io
```

Comment:
This is an ApiGroup not a resource and needs to be removed

Reply:
Done